Breeding-Insight/bi-web
# [BI-1867] - Ontology: Increase name length to 16 characters

# Description
**Story:**  [ BI-1867 - Ontology: Increase name length to 16 characters ](https://breedinginsight.atlassian.net/browse/BI-1867)

Modified front end logic to increase ontology name length limit from 12 to 16 characters.

# Dependencies
[bi-api: feature/BI-1867](https://github.com/Breeding-Insight/bi-api/pull/317)

# Testing
In both ontology import and manual entry of ontology:
- Try to add an ontology term with 16 characters, should succeed and new entry show up in ontology table 
- Try to add an ontology term with 17 characters, confirm that an error message indicating that character limit must be 16 or less shows up and import is blocked and no new terms are added to ontology table

# Checklist:
